### PR TITLE
7132 conference details

### PR DESCRIPTION
**Pull request by kris45**

### CODE REVIEW

File: conference/react-js/src/components/Tabs/index.jsx [hunk 1]
```diff
@@ -5,9 +5,13 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Upcoming from '../Upcoming';
 import AutorizedUserMenu from '../Autorized-user-menu';
-import { upcoming, myTalks, myEvents } from '../../constants/route-url';
+import { upcoming,
```
Comment (ScreamCat1) [inline conference/react-js/src/components/Tabs/index.jsx]: Maybe, you should replace upcoming to new line

File: conference/react-js/src/containers/Conference/index.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  start_date: PropTypes.string,
+  title: PropTypes.string,
+  id: PropTypes.number,
+}) };
+
+Conference.defaultProps = {
+  conference: {},
+};
+
+const mapStateToProps = ({ conference }) => ({ conference });
```
Comment (ScreamCat1) [inline conference/react-js/src/containers/Conference/index.jsx]: Maybe, you can write like this "const mapStateToProps = conference => conference  "

**Review by ScreamCat1 — APPROVED** [review]

File: conference/react-js/src/containers/Conference/index.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Talks from '../Talks';
+
+const Conference = ({ conference: {
+  call_for_paper_end_date: callForPaperEndDate,
```
Comment (blyzniuk) [inline conference/react-js/src/containers/Conference/index.jsx:7]: Why ```callForPaperEndDate``` but ```CallForPaperStartDate``` ? 
What logic of capital letter?

File: conference/react-js/src/components/Card/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
+import React, { PureComponent } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { conference } from '../../constants/route-url';
+
+class Card extends PureComponent {
+setConferenceData = () => {
```
Comment (blyzniuk) [inline conference/react-js/src/components/Card/Card.jsx]: indentation

**Review by mmelnik — APPROVED** [review]

**Pull request merged**